src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import { FC, useState, useEffect } from 'react';
+import cn from 'classnames';
+
+import { getUser } from '../../api';
+
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  todo: Todo;
+  onClose: () => void;
+}
+
+export const TodoModal: FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { id, title, completed, userId } = todo;
+
+  const handleClose = () => {
+    onClose();
```

Comment:
The `handleClose` method is redundant here. You can directly use the `onClose` prop in your `onClick` event handler.